pkg/analyser/analyzer.go

```diff
@@ -6,9 +6,10 @@ import (
 
 	resourceaws "github.com/cloudskiff/driftctl/pkg/resource/aws"
 
+	"github.com/r3labs/diff/v2"
```

Comment:
pretty sure you have something different to handle import sorting we need to check that together